Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,110 @@
+Code Review Etiquette
+===========
+
+This document describes a general etiquette and code of conduct that every team member should follow, as well as some concrete guidelines and activities we use in the team related to code reviews.
+The purpose of these guidelines is to formalise some basic requirements for pull requests and receiving/providing feedback to improve the quality of code reviews and teamwork in general.
+
+Note: Guidelines in this document should be considered to be of equal importance
+
+Note: This document is heavily inspired by [this blog post](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)
+
+Everyone
+--------
+
+* Be respectful to each other
+* Be humble. ("I'm not sure - let's look it up.")
+* Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+* Don't use sarcasm or trolling
+* If you feel annoyed at something/someone, don't make it affect your or others' work. Talk to a friend, go for a walk, meditate, switch to something else.
+* Talk synchronously, e.g. call or in person, if there are too many "I didn't understand" or "Alternative solution:" comments. Prefer not to use chat for that. Post a follow-up
+  comment summarizing the discussion.
+* Avoid selective ownership of code ("mine", "not mine", "yours"). Prefer referring to it as `our code`.
+* Use "we" instead of "you", unless asking for opinion of specific individual ("What do you think ...?", but "Here we are doing ... What if we ...?",)
+* Avoid using terms that could be seen as referring to personal traits. ("dumb",
+  "stupid", "insane"). Assume everyone is intelligent and well-meaning.
+* Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Can you use positive language as opposed to neutral?
+* When using emojis be aware that their meaning can be different for different individuals - winking smile may not be always innocent
+* Same with any other media format you decide to use
+* Do not expect your comments to be answered immediately. If you need an answer fast - talk in person (the same applies to the Slack messages)
+* It's fine to have disagreements and it's hard to please everyone.
+
+
+Creating a Pull Request
+--------
+
+* When creating your branches, name them using a format like `<author>/<ticket>` or `<author>/<ticket>-<shortdescription>`
```

Comment:
I wonder if this could be automated with `Steve` e.g when a person moves ticket in progress we could create branch if need with correct format. What do you guys think?

Reply:
you mean listen to JIRA webhooks? Interesting idea. But we can start with something simple like  a git hook maybe.

Reply:
Nice idea @sergdort